x/posts/internal/keeper/keeper_test.go

```diff
@@ -400,7 +400,7 @@ func TestKeeper_RemoveReaction(t *testing.T) {
 			expectedStored: types.Reactions{},
 		},
 		{
-			name:           "Non existing like returned expError - Reaction",
+			name:           "Non existing like returned expError - Value",
```

Comment:
Why has this changed?

```suggestion
			name:           "Non existing reaction returns error - Value",
```